lib/Mojolicious/Guides/Tutorial.pod

```diff
@@ -342,6 +342,37 @@ actions to templates.
 A list of all built-in ones can be found in
 L<Mojolicious::Plugin::DefaultHelpers> and L<Mojolicious::Plugin::TagHelpers>.
 
+=head2 Plugins
+
+Plugins are application extensions that help with code sharing and
+organization. You can load a plugin with the keyword
+L<Mojolicious::Lite/"plugin">, which can omit the C<Mojolicious::Plugin::> part
+of the name, and optionally provide configuration for the plugin.
+
+  use Mojolicious::Lite;
+
+  plugin Config => {file => '/etc/myapp.conf', default => {foo => 'bar'}};
+
+  # Return configured foo value, or default if no configuration file
+  get '/foo' => sub {
+    my $c   = shift;
+    my $foo = $c->app->config('foo');
+    $c->render(json => {foo => $foo});
+  };
+
+  app->start;
+
+Some built-in plugins include L<Mojolicious::Plugin::DefaultHelpers>, which
+adds many helpers and is loaded automatically, and
+L<Mojolicious::Plugin::Config>, which can populate L<Mojolicious/"config">
+using a config file. Plugins can also set up routes, hooks, handlers, or even
+load other plugins.
+
+A list of built-in plugins can be found at L<Mojolicious::Plugins/"PLUGINS">.
+Many more plugins are available from CPAN, and you can even write plugins
+specifically to help organize your application as described at
+L<Mojolicious::Guides::Cookbook/"Adding a plugin to your appplication">.
```

Comment:
This is too verbose for a tutorial section. Just replace all of this with a link to [MetaCPAN](https://metacpan.org/requires/distribution/Mojolicious), like we have on the frontpage.